content/en/data_jobs/airflow.md

```diff
@@ -139,8 +146,11 @@ For Astronomer customers using Astro, <a href=https://www.astronomer.io/docs/lea
    openlineage-python>=1.23.0
    ```
 
-2. Configure the OpenLineage provider. You can do this by setting the following environment variables [using the Astro UI][5]:
-   
+2. To set up the OpenLineage provider, define the following environment variables. You can configure these variables in your Astronomer deployment using either of the following methods:
+
+  - [From the Astro UI][5]: Navigate to your deployment settings and add the environment variables directly.
```

Comment:
Can you please select all the content under this numbered list item (lines 151-170) and indent them by one space so that it all lines up under #2?